Xamarin.Forms.Platform.iOS/Renderers/MediaElementRenderer.cs

```diff
@@ -24,9 +24,7 @@ public sealed class MediaElementRenderer : ViewRenderer<MediaElement, UIView>
 		[Internals.Preserve(Conditional = true)]
 		public MediaElementRenderer()
 		{
-			Xamarin.Forms.MediaElement.VerifyMediaElementFlagEnabled(nameof(MediaElementRenderer));
-
-			_playedToEndObserver = NSNotificationCenter.DefaultCenter.AddObserver(AVPlayerItem.DidPlayToEndTimeNotification, PlayedToEnd);
```

Comment:
We already add the observer here https://github.com/xamarin/Xamarin.Forms/blob/7c7af1356dba6ae0a8ce89ee161d1db8de454036/Xamarin.Forms.Platform.iOS/Renderers/MediaElementRenderer.cs#L439